nvflare/fuel/hci/proto.py

```diff
@@ -107,27 +118,29 @@ def append_table(self, headers: List[str], name=None) -> Table:
         self.data.append({ProtoKey.TYPE: ProtoKey.TABLE, ProtoKey.ROWS: t.rows})
         return t
 
-    def append_string(self, data: str, meta: dict = None):
-        self.data.append({ProtoKey.TYPE: ProtoKey.STRING, ProtoKey.DATA: data})
+    def add_meta(self, meta: dict):
```

Comment:
minor personal preference: "update_meta"

Reply:
update_meta seems to be better.

Reply:
renamed to update_meta